modules/dr-restoring-cluster-state.adoc

```diff
@@ -760,7 +760,7 @@ etcd-ip-10-0-154-194.ec2.internal                2/2     Running     0
 etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0          9h
 ----
 
-To ensure that all workloads return to normal operation following a recovery procedure, restart each pod that stores `kube-apiserver` information. This includes {product-title} components such as routers, Operators, and third-party components.
+To ensure that all workloads return to normal operation following a recovery procedure, restart all control plane nodes and worker nodes.
```

Comment:
As my understanding, worker nodes need not be restarted, is it make sense?

Reply:
@geliu2016 I updated line 763 to state that only control plane nodes need to be restarted. Thanks!